tests/widgets_paragraph.rs

```diff
@@ -142,11 +144,68 @@ const SAMPLE_STRING: &str = "The library is based on the principle of immediate
      interactive UI, this may introduce overhead for highly dynamic content.";
 
 #[test]
-fn widgets_paragraph_can_wrap_its_content() {
+fn widgets_paragraph_can_char_wrap_its_content() {
```

Comment:
Only as an observation only (and not something worth changing), when modifying existing tests, and adding new tests in the same PR, it can make it easier to read the diffs if you write the new tests after the existing tests rather than before. This makes the diff show the new code as added, rather than showing it the new code as a modification of the existing code and then showing the existing code as additions.

It's a super minor thing, but it helps when reviewing the code to be able to see that generally existing tests are unchanged (which means that the tested behavior is stable).

Reply:
I wrote it this way because I believe `CharBoundary` should be the default wrapping mode, and I'm a bit OCD so I like to organize tests based on that.

Reply:
yup - I would have done the same :D